### PR TITLE
Help/About: Initial pass of 6.2 About page

### PR DESCRIPTION
**Pull request by ryelle**
This is a first pass at getting the About strings in, updating the header assets, and updating the style and colors. It sounds like the content is ready-enough to commit with translation wrappers, so I've done that, but some things will likely change in the time before RC1.

I'd appreciate a review from @jpantani or @thetinyl to make sure I've copied over the content correctly — you can review the screenshot below if you don't have a local environment set up.

**Screenshots:**

About page (full page)

about-full

Credits (only header changed)

credits

Freedoms (only header changed)

freedoms

Privacy (only header changed)

privacy

Trac ticket: https://core.trac.wordpress.org/ticket/57477

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by thetinyl** [comment]
@ryelle I think it looks good. I'm noticing two updates needed:

- Remove the headline above the Style Book, Copy and paste, Custom CSS section (this change is now reflected in the copy document, too)
- The "Learn more about 6.2" bottom section needs a little bit of tweaking—see screenshot below from Figma design:

image

**Comment by ryelle** [comment]
@thetinyl Is the difference in the "Learn more" section the vertical alignment? I did leave all the strings here alone, since this way the current translations can be reused (they've been consistent for the past few releases, I think), but if the copy change is also important I can do that too.

Edit: Ah, I missed the icons, too. Still have the question about the copy.

**Comment by thetinyl** [comment]
> I did leave all the strings here alone, since this way the current translations can be reused (they've been consistent for the past few releases, I think), but if the copy change is also important I can do that too.

The copy is actually very similar to what it's been, just condensed a little. It was adjusted to make the resource links stand out a little better. Though from a translation perspective it's probably different enough that it would probably need another look.

**Comment by ryelle** [comment]
I've removed the heading, fixed the alignment, and added in the icons. The bottom section now looks like this:

Screen Shot 2023-02-28 at 23 33 58

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -53,429 +53,155 @@
...
-					printf(
-						/* translators: %s: "General Settings" admin page title, linked to the page if the user can edit options. */
-						__( 'The site tagline is empty by default in new sites but can be modified in %s.' ),
-						current_user_can( 'manage_options' ) ? '<a href="' . esc_url( admin_url( 'options-general.php' ) ) . '">' . __( 'General Settings' ) . '</a>' : __( 'General Settings' )
-					);
-					?>
-				</p>
-				<p><?php _e( 'A new modal design offers a background blur effect, making it easier to focus on the task at hand.' ); ?></p>
+				<h3><?php _e( 'Experience the Site Editor, now out of beta' ); ?></h3>
+				<p><?php _e( 'Stable and ready for you to dive in and explore: 6.2 is your personal invitation to discover what the next generation of WordPress—and Block themes—can do.' ); ?></p>
```
Comment (audrasjb) [inline src/wp-admin/about.php:147]: Maybe some missing spaces here?
`WordPress — and Block themes — can do`

Comment (audrasjb) [reply src/wp-admin/about.php:147]: well, no it looks good in en_US
https://prowritingaid.com/grammar/1000066/Should-I-use-spaces-around-an-em-dash

**Review by audrasjb — APPROVED** [review]

File: src/wp-admin/about.php (hunk 1 above)
Comment (ryelle) [reply src/wp-admin/about.php:147]: Looks like without spaces is part of the WP brand style: ["Use an em dash (—) without spaces."](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/#punctuation)

**Comment by richtabor** [comment]
LGTM. We can iterate from here.

**Comment by ryelle** [comment]
Committed in [r55449](https://core.trac.wordpress.org/changeset/55449).

**Pull request closed without merging**